pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -162,6 +162,15 @@ func (r *PyTorchJobReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, err
 	}
 
+	t, err := util.DurationUntilExpireTime(&pytorchjob.Spec.RunPolicy, pytorchjob.Status)
+	if err != nil {
+		logrus.Warnf("Reconcile PyTorchJob error %v", err)
```

Comment:
We are using mix of logger and logrus in code. Need to cleanup in a separate PR.

Reply:
Oh, exactly.